api/cpp/tests/manual/platform_qt/main.cpp

```diff
@@ -49,18 +49,20 @@ static slint_platform::NativeWindowHandle window_handle_for_qt_window(QWindow *w
 #endif
 }
 
-class MyWindow : public QWindow, public slint_platform::WindowAdapter<slint_platform::SkiaRenderer>
+class MyWindow : public QWindow, public slint_platform::WindowAdapter
 {
+    std::unique_ptr<slint_platform::SkiaRenderer> m_renderer;
```

Comment:
this doesn't need to be a unique_ptr, does it?  you can easily construct it in the constructor.

Reply:
"Easily" means bringing back the dance of initialing the field after calling `QWindow::create()` on the base class and creating the native window handle. That's the dance I'd like to avoid :)